tuning/sft_trainer.py

```diff
@@ -248,6 +277,11 @@ def train(
             "Validation dataset length is %s", len(formatted_validation_dataset)
         )
 
+    if framework is not None and framework.requires_agumentation:
+        model, (peft_config,) = framework.augmentation(
+            model, train_args, modifiable_args=(peft_config,)
```

Comment:
@fabianlim what will happen if `peft_config` is Ptconfig or None? only LoraConfig is supported by the framework currently?

Reply:
some of the `augmentation` requires some of the existing configurations to operate properly. For instance GPTQ LoRA, we need to know the lora configuration to instantiate the LoRA trainable model. 

So for instance inside our GPTQ plugin, we need to check if `peft_config` is None, if it is we will [raise an assertion](https://github.com/foundation-model-stack/fms-acceleration/blob/79bc89b3ce15d804e826d7c98e11bde0182fa7a4/plugins/accelerated-peft/src/fms_acceleration_peft/framework_plugin_autogptq.py#L205).

Reply:
I have updated the PR starting from https://github.com/foundation-model-stack/fms-hf-tuning/pull/157#:~:text=%E2%80%A6-,9580b99,-fmt%20%2B%20lint to remove the yamls.